build.gradle

```diff
@@ -45,46 +46,54 @@ dokkaHtml.configure {
     }
 }
 
-publishing {
-    publications {
-        create("default", MavenPublication) {
-            from(components["java"])
+android {
+    compileSdkVersion 30
+}
+
+afterEvaluate {
```

Comment:
You might want to ignore whitespace changes when looking at this. Otherwise you might miss the actual changes I made, besides wrapping `publishing { ... }` around `afterEvaluate { ... }`.

![whitespace](https://user-images.githubusercontent.com/369863/110159382-8a38ef00-7de2-11eb-8e89-4bb8bb54aeeb.png)